### ISSUE TITLE
No devices found on multi-NIC setup

### ISSUE THREAD

**Issue by cleesieboy** [duncan3dc/sonos#43]
Hi, trying to get this up and running on my (linux Debian) server that has two NIC's (one for my internal LAN and one for external WAN). I get the "No devices found on the current network" message. Forcing all SSDP calls to a specific NIC by setting up a static route solves the issue, but it'd be preferable if this could be handled directly from the API. Or am I going about this the wrong way?

**Comment by duncan3dc** [comment]
That sounds like something the library should handle, can you test out a development version for me to see if it solves your problem?
```sh
composer require duncan3dc/sonos:dev-custom-network-interface
```
```php
$sonos = new Network;

# See the documentation on IP_MULTICAST_IF at http://php.net/manual/en/function.socket-get-option.php
$sonos->setNetworkInterface("eth0");
```

**Comment by cleesieboy** [comment]
After realising I had to delete the cache, that indeed did the trick! Great work! :)

**Comment by duncan3dc** [comment]
Sorry for not mentioning the cache. The [setNetworkInterface()](https://duncan3dc.github.io/sonos/setup/#ssdp-discovery) method is available in [1.8.0](https://github.com/duncan3dc/sonos/releases/tag/1.8.0) now
